kbn_pm/src/commands/bootstrap/bootstrap_command.mjs

```diff
@@ -98,30 +83,27 @@ export const command = {
       }),
     ]);
 
-    // Bootstrap process for Bazel packages
-    // Bazel is now managing dependencies so yarn install
-    // will happen as part of this
-    //
-    // NOTE: Bazel projects will be introduced incrementally
-    // And should begin from the ones with none dependencies forward.
-    // That way non bazel projects could depend on bazel projects but not the other way around
-    // That is only intended during the migration process while non Bazel projects are not removed at all.
-    if (forceInstall) {
-      await time('force install dependencies', async () => {
-        await removeYarnIntegrityFileIfExists();
-        await Bazel.expungeCache(log, { quiet });
-        await Bazel.installYarnDeps(log, { offline, quiet });
+    if (shouldInstall) {
+      await time('install dependencies', async () => {
+        if (forceInstall) {
+          await removeYarnIntegrityFileIfExists();
+        }
+        log.info('installing dependencies with yarn');
```

Comment:
I just realised we have `installing dependencies with yarn` and `installing node dependencies with yarn` which I think it's somewhat confusing in the bootstrap logs

Reply:
right, thanks, it's a duplicate! :)